editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -39,8 +49,20 @@ function Component(props: Props) {
             <Typography fontWeight={FONT_WEIGHT_BOLD}>
               Required files
             </Typography>
-            {props.fileTypes.map((fileType) => (
-              <p key={fileType.key}>{fileType.key}</p>
+            {fileList.required.map((file) => (
+              <p key={file.key}>{file.key}</p>
+            ))}
+            <Typography fontWeight={FONT_WEIGHT_BOLD}>
+              Recommended files
+            </Typography>
+            {fileList.recommended.map((file) => (
+              <p key={file.key}>{file.key}</p>
+            ))}
+            <Typography fontWeight={FONT_WEIGHT_BOLD}>
+              Optional files
+            </Typography>
+            {fileList.optional.map((file) => (
+              <p key={file.key}>{file.key}</p>
```

Comment:
I'm not actually sure if we need these here based on the Figma wireframes, but they will be needed in the modal.

Reply:
Guessing the `optional` list will be particularly long (especially if it needs to include more than the explicitly configured `Not required` files per my other question), so best contained to the modal makes sense to me :ok_hand:

Reply:
Cool this matches what I think also so I've now dropped this from here.